Sources/Views/MessageInputBar.swift

```diff
@@ -252,14 +252,14 @@ open class MessageInputBar: UIView {
         textViewHeightAnchor = inputTextView.heightAnchor.constraint(equalToConstant: maxHeight)
         
         leftStackViewLayoutSet = NSLayoutConstraintSet(
-            top:    inputTextView.topAnchor.constraint(equalTo: topAnchor, constant: padding.top),
+            top:    leftStackView.topAnchor.constraint(equalTo: topAnchor, constant: padding.top),
```

Comment:
Because we are setting a topAnchor we need the default `InputStackView` alignment property to be bottom so the buttons don't remain in the center when the textView grows

Reply:
@nathantannar4 I updated the requested changes.:beers::beers::beers: